pulseapi/profiles/admin.py

```diff
@@ -0,0 +1,19 @@
+from django.contrib import admin
+from .models import UserProfile, UserBookmarks
+
+class UserProfileAdmin(admin.ModelAdmin):
+    """
+    Show a list of entries a user has submitted in the EmailUser Admin app
```

Comment:
we will, but that doesn't make this the right comment - fixing to say it shows the profile-associated user, if there is one set.